lib/widgets/action_sheet.dart

```diff
@@ -60,6 +71,49 @@ abstract class MessageActionSheetMenuItemButton extends StatelessWidget {
   }
 }
 
+// This button is very temporary, to complete #125 before we have a way to
+// choose an arbitrary reaction (#388). So, skipping tests and i18n.
+class AddThumbsUpButton extends MessageActionSheetMenuItemButton {
```

Comment:
I think it wouldn't take much work to write just a test or two for this, so it's worth doing that. The main thing for a test is: tap the icon, check that we made an appropriate API request.

A bonus test would be: prepare an error response, tap the icon, check that there's an error dialog (and so we didn't just crash).

Reply:
Can drop the "skipping tests" part now :-)